File: 3st/Juram/programmer/중앙값 구하기.js
function solution(array) {
    var answer = 0;
    array.sort((a,b)=>a-b);
    if(array.length % 2 != 0){
        return array[Math.floor(array.length/2)]
    }else{
        return (array[parseInt(array.length/2-1)+1]+array[parseInt(array.length/2)])/2;
    }
    return answer;
}

File: 4st/Seohyun/programmer/중앙값 구하기.java
class Solution {
    public int solution(int[] array) {
        int answer = 0;
        int temp=0;
        for(int i=1; i<array.length; i++) {
            for(int j=0; j<i; j++) {
                if(array[i]<array[j]) {
                    temp=array[i];
                    array[i]=array[j];
                    array[j]=temp;
                }
            }
        }
        return answer=array[(array.length+1)/2-1];
    }
}